doc/user/content/sql/create-sink.md

```diff
@@ -46,10 +47,19 @@ Field | Use
 **TOPIC** _topic&lowbar;prefix_ | The prefix used to generate the Kafka topic name to create and write to.
 **KEY (** _key&lowbar;column&lowbar;list_ **)** | An optional list of columns to use for the Kafka key. If unspecified, the Kafka key is left unset. {{< version-added v0.5.1 />}}
 **CONSISTENCY TOPIC** _consistency&lowbar;topic_ | Makes the sink emit additional [consistency metadata](#consistency-metadata) to the named topic. Only valid for Kafka sinks. If `reuse_topic` is `true`, a default consistency_topic will be used when not explicitly set. The default consistency topic name is formed by appending `-consistency` to the output topic name. {{< version-added v0.8.4 />}}
-**CONSISTENCY FORMAT** _format_ | The format of the Kafka consistency topic. Defaults to the format of the sink if not provided. {{< version-added v0.8.4 />}}
-**WITH OPTIONS (** _option&lowbar;_ **)** | Options affecting sink creation. For more details see [`WITH` options](#with-options).
+**CONSISTENCY FORMAT** _consistency&lowbar;format&lowbar;spec_ | The format of the Kafka consistency topic. Defaults to the format of the sink if not provided. For more detail, see [Consistency `FORMAT`](#consistency-format). {{< version-added v0.8.4 />}}
+**WITH (** _option&lowbar;list_ **)** | Options affecting Materialize's connection to Kafka. For more detail, see [Kafka connector `WITH` options](#kafka-connector-with-options).
 **CONFLUENT SCHEMA REGISTRY** _url_ | The URL of the Confluent schema registry to get schema information from.
 
+### Kafka connector `WITH` options
+
+The following options are valid within the Kafka connector's `WITH` clause.
+
+Field                | Value type | Description
+---------------------|------------|------------
+`username `          | `text`     | The Kafka username.
+`password `          | `text`     | The Kafka password.
```

Comment:
Is this right? I'd expect `sasl_username` and `sasl_password`. And that already seems to be covered by the `WITH` option sections below. Is this meant to be for the CSR `WITH` options?

Reply:
I think these are the params to configure HTTP basic auth for the schema registry? At least that's the only time I can remember using bare username/password WITH options.

Reply:
The only function to read from the inner `WITH` options is [`generate_ccsr_client_config`](https://github.com/MaterializeInc/materialize/blob/main/src/sql/src/kafka_util.rs#L487), which reads `username` and `password`.